app/views/claims/address.html.erb

```diff
@@ -44,7 +44,7 @@
             County
           <% end %>
           <%= errors_tag current_claim, :address_line_4 %>
-          <%= form.text_field :address_line_4, autocomplete: "region",
+          <%= form.text_field :address_line_4, autocomplete: "address-level1",
```

Comment:
I'm gonna be picky here. I don't think we need this level of address. A postal address only needs a street address, post town and postcode to be valid (a locality (e.g. Village, small town etc) can be added too). A county isn't part of the address specification that the Royal Mail use, so we can probably ding it. (I can bore on about addresses for hours)

Reply:
It also adds added complexity, where the autocomplete value here might be the town for some users

Reply:
Interesting! You know your addresses! :) Are you saying we loose the field entirely or just the autocomplete attribute?

For me, this story was only about resolving the html parser issues @titlescreen raised, I would be wary of changing anything else.

Reply:
Yeah, I'd get rid of the autocomplete for that one, and probably use `street-address`, `address-line1` etc, rather than the administrative level ones, because they're a bit unclear.